lib/chain.js

```diff
@@ -11,6 +11,7 @@ const async = op => {
     case 'series': return series;
     case 'find': return find;
   }
+  return null;
```

Comment:
```js
return (items, fn, callback) => {
  return callback(null, items.map(id));
};
```
where
`const Id = x => x;`

Reply:
Why not simple
`items.map(id)` -> `items.slice()`?

Reply:
Yes, `.slice(0)` will be better